flycheck.el

```diff
@@ -9350,6 +9350,12 @@ See URL `http://pear.php.net/package/PHP_CodeSniffer/'."
             ;; been configured with show_progress enabled
             "-q"
             (option "--standard=" flycheck-phpcs-standard concat)
+            ;; Some files are not detected correctly
+            ;; so it is necessary to pass the extension.
+            (eval
+             (-when-let (ext  buffer-file-name fn (file-name-extension buffer-file-name))
```

Comment:
Did you test this new code? It sets the variable `ext` to the value of `buffer-file-name`

Reply:
yes, i test it with the scratch buffer, file without extension and a file with extension.
No exception throw

Reply:
The syntax on the line with the -when-let is wrong: the fn part and the file-name-extension part are ignored.  Can you fix it? I gave you the correct syntax in my first comment.

Reply:
Ok, seems like now with -when-let* it's working